resources/cdk/photo_asset_manager/README.md

```diff
@@ -1,13 +1,13 @@
-# PAM CDK
+# AWS CDK for PAM
 
-This project will create the following in your AWS cloud environment:
+This project will create the following in your AWS Cloud environment:
 
-- IAM group
+- AWS Identity and Access Management (IAM) group
 - IAM user (added to the IAM group)
-- S3 buckets
-- DynamoDB tables
-- Lambda function that performs image classification via AWS Rekognition when new images are uploaded to the S3 bucket
-- API Gateway routing for lambda functions
+- Amazon Simple Storage Service (Amazon S3) buckets
+- Amazon DynamoDB tables
+- AWS Lambda function that performs image classification with Amazon Rekognition when new images are uploaded to the S3 bucket
+- Amazon API Gateway routing for Lambda functions
```

Comment:
Using the "first appearance" names really reduces the readability, but I guess we can take it up with Marketing. 

As a reader, you're testing my patience by making me read out AWS Simple Storage Service and Amazon Identity Access Management.

Could we compromise with a hyperlink to the product page for each and keep the readable names? (IAM, S3)

Reply:
I sort of agree? Or at least I did at first. The more I thought about it though the more I realized this could be the first time the customer has every come across a term like API Gateway, and they might be unaware it's an Amazon product.

Reply:
That's certainly the policy the docs org has taken for editing these documents. This isn't the first time anyone has seen these types of changes come through :)